admin/class-calibration-beta.php

```diff
@@ -0,0 +1,141 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ * @since   9.3.0
+ */
+
+/**
+ * Holds the logic for the calibration beta.
+ */
+class WPSEO_Calibration_Beta implements WPSEO_WordPress_Integration {
+
+	/**
+	 * @var string Name of the options.
+	 */
+	protected $option_name = 'recalibration_beta';
+
+	/**
+	 * Shows the feature toggle.
+	 *
+	 * @return void
+	 */
+	public function show_feature_toggle() {
```

Comment:
If this is a one-time usage and will be removed after beta, I'd say it's fine to temporarily duplicate all this code. Otherwise, if it's going to stay, I'd say `toggle_switch()` should be modified to allow printing some content before and after the toggle instead of the Help button.

Reply:
It's for one-time usage indeed 😄